Misc/NEWS.d/next/Library/2024-07-31-17-22-10.gh-issue-83938.TtUa-c.rst

```diff
@@ -0,0 +1,3 @@
+Stop incorrectly using RFC 2047 "encoded words" for email addresses with
+non-ASCII characters when email.generator is called using a policy with
+``utf8=False``.
```

Comment:
This needs to mention the new error that will be raised.  Here is my suggestion:

The :mod:`email` module no longer incorrectly encodes non-ascii characters in email addresses using :rfc:2047 encoding.  Under a policy with `utf8=True` this means the addresses will be correctly passed through.  Under a policy with `utf8=False`, attempting to serialize a message with non-ASCII email addresses will now result in a :exc:`InvalidMailboxError`.